my_dual_robot_cell/my_dual_robot_cell_control/urdf/my_dual_robot_cell_controlled.urdf.xacro

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<robot name="my_dual_robot_cell" xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <!--First, include the macro from our custom description-->
+  <xacro:include
+    filename="$(find my_dual_robot_cell_description)/urdf/my_dual_robot_cell_macro.xacro" />
+  <!--The driver already provides a control tag for the UR arm, we can directly include-->
+  <xacro:include filename="$(find ur_robot_driver)/urdf/ur.ros2_control.xacro" />
```

Comment:
Hey shoudnt this be :
```xml
<xacro:include filename="$(find ur_description)/urdf/ur.ros2_control.xacro" />
```
as the urdf is in the description package not the driver